### ISSUE TITLE
Part 4: Step 14

### ISSUE THREAD

**Issue by TheAquaSheep** [FlimFlam69/WiiUTutorial#23]
Part 4: Step 14 claims that the wii u will black screen after redNAND has finished installing.
My experience was that the wii u simply rebooted into normal sysNAND.
I expected what I observed as Haxchi was installed by this point but not as cold boot.

I also suggest adding a step about backing up the redNAND to computer here using software that can read the hidden partition.
I used sdio NAND manager, which another tutorial mentioned, to back up SLP, MLP, and SLCMPT at the same time as restoring my sd card contents.

NB - OTP and SEEPROM were not dumped at this stage as they are only created the first time Mocha CFW attempts to use them with the appropriate redirections enabled, which cannot happen until the SD card contents have been restored and Mocha CFW boots into redNAND at least once. After this, the OTP and SEEPROM files can be extracted from the SD card.

As a completely separate, and minor, issue, after following Part 0: some apps, mocha and disc2app from memory, were functional but lacking xml and png files that allow them to be recognised as installed by the homebrew app store. Simply choosing to install them via the app store remedied this.

**Comment by JohnVeness** [comment]
I agree with all of this. For me too I got no black screen after setting up rednand.
